src/app/builders/components/Container/ActionMetricsContainer.tsx

```diff
@@ -0,0 +1,20 @@
+import { cn } from '@/lib/utils'
+import { FC, PropsWithChildren } from 'react'
+import { CommonComponentProps } from '../commonProps'
+
+export type ActionMetricsContainerProps = CommonComponentProps & PropsWithChildren
+
+export const ActionMetricsContainer: FC<ActionMetricsContainerProps> = ({
```

Comment:
What is the difference between this component and the one in `Backers`? If there is none, I would reuse it, remember that we can just call components from parents components.